logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/RetryFailedMessageUseCase.kt

```diff
@@ -57,23 +57,23 @@ class RetryFailedMessageUseCase internal constructor(
 
     /**
      * Function that enables resending of failed message to a given conversation with the strategy of fire & forget.
-     * This message must have a status of [Message.Status.FAILED].
+     * This message must have a status of [Message.Status.Failed] or [Message.Status.FailedRemotely].
      *
      * If it's an asset message, the asset may or may not be already uploaded. The asset will be uploaded if needed.
      *
-     * The resending and possible reuploading of assets are scheduled but not awaited, so returning [Either.Right] doesn't mean that
+     * The resending and possible re-uploading of assets are scheduled but not awaited, so returning [Either.Right] doesn't mean that
```

Comment:
It was actually correctly spelled before.

The `re` prefix is used without hyphens when adding to a verb, like in `retry`, `redo`, etc.

It's only used with hyphen when the word could be confused, like `reserve` and `re-serve` (making a reservation _vs_ serving something again), or `re-cover` and `recover` (cover again `vs` getting well).

Reply:
refixed 😆